WIP:Add riops forecast dataset

racetted opened the pull request:
First draft, still need to review some of the parameters. 

Contains a fix to the test.yml file for a bad url.

racetted committed:
test.yml -- fix topic hierarchy url

racetted committed:
riops forecast first draft

URL and spatial extents still WIP

tomkralidis commented on metadata/mcf/msc_nwp_riopsf.yml:
```diff
@@ -0,0 +1,65 @@
+              resolution: P3H
+    url: https://met5node.weather.gc.ca/catalogue
+
+distribution:
```
```yaml
type: application/geo+json
rel: items
```

tomkralidis commented on metadata/mcf/msc_nwp_riopsf.yml:
```diff
@@ -0,0 +1,65 @@
+metadata:
+    charset: utf8
+    hierarchylevel: dataset
+    identifier: urn:wmo:md:${MET5_CENTRE_ID}:ocean:medium-range:nwp.riops
```
`s/MET5_CENTRE_ID/WIS2_CENTRE_ID/g`

racetted replied:
Intentional change, didn't think we'd reuse the exact same ones, but I can be mistaken.

tomkralidis replied:
We are using the identical centre id as WIS2 proper, yes.

racetted replied:
Ok, noticed your github action, adjusting.

racetted committed:
Adjusted riopsf yml

racetted committed:
Fix to base mcf in riops

racetted committed:
riopsf - added license for recommended

racetted committed:
riops - adjusted WIS2 topic

racetted committed:
riops - adjusted topic again

racetted committed:
riopsf - wis2 topic change to pass validation

met5 not yet recognized

racetted committed:
Added debug in github action

racetted committed:
test.yml - fix topic structure

tomkralidis committed:
Update test.yml

racetted committed:
update riopsf dataset

Updated the bounding boxes and projection statements after reviewing the
files.

racetted committed:
Added riops analysis

racetted committed:
Update riops

Adjusted forecast period and analysis categorization.
